src/vs/workbench/contrib/extensions/browser/extensionsViewlet.ts

```diff
@@ -645,6 +652,11 @@ export class ExtensionsViewPaneContainer extends ViewPaneContainer implements IE
 		this.hasInstalledExtensionsContextKey.set(result.some(r => !r.isBuiltin));
 	}
 
+	private async updateOutdatedExtensionsContexts() {
+		const result = await this.extensionsWorkbenchService.queryLocal();
+		this.hasOutdatedExtensionsContextKey.set(result.some(r => !r.outdated));
```

Comment:
I do not think we need to set this context, because it is already being updated here - 

https://github.com/microsoft/vscode/blob/3ad265eb02ad18c090c85a1fee21b673be196186/src/vs/workbench/contrib/extensions/browser/extensionsWorkbenchService.ts#L1566-L1570

Reply:
You're right, I didn't see that